src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,63 @@
+package duke;
+
+import duke.tasks.Task;
+
+import java.util.Scanner;
+
+public class Ui {
+    public static String HELLO = "Hello! I am Duke Nice To Meet You\n";
+    public static String BYE = "Bye! Hope to See You Again!";
+    public static String ADD = "Got it fam! I've added this task:\n ";
+
+
+    private Scanner sc;
+    public Ui() {
+        this.sc = new Scanner(System.in);
+    }
+
+    public String getCommand() {
+        return sc.nextLine();
+    }
+
+    public void showError(String e_message) {
+        System.out.println(e_message);
+    }
+
+    public void greet() {
+        System.out.println(HELLO);
+    }
+
+    public void bye() {
```

Comment:
May I clarify the intention behind the naming of this function? Perhaps you could consider `printBye()` instead, so as to convey the action performed by the function.